src/main/java/com/uid2/admin/AdminConst.java

```diff
@@ -2,6 +2,7 @@
 
 public class AdminConst {
     public static String enableKeysetConfigProp = "enable_keysets";
+    public static final String throwOnS3WriteException = "throw_on_s3_write_exception";
```

Comment:
This isn't really about S3 write exceptions, but more about exception during backfilling. We would still error on other S3 writes by admin.  

Also naming is inconsistent with existing conventions for naming constants (ALL_CAPS_WITH_UNDERSCORES).

Reply:
I don't think we need a feature switch for this at all 🤷